pkg/util/pem.go

```diff
@@ -87,7 +88,10 @@ func ValidateAndSplitPEMBundle(data []byte) ([][]byte, error) {
 			return nil, fmt.Errorf("invalid PEM block in bundle; invalid PEM certificate: %w", err)
 		}
 
-		certificates = append(certificates, pem.EncodeToMemory(b))
+		// check the duplicate certificates
+		if !certPool.IsCertificateDuplicate(b.Bytes) {
```

Comment:
Some changes missing here? I think the idea was to use the new certPool type.

Reply:
Ah, sure! I didn't recognize your requirement properly!
I'm going to fix the code.

Reply:
Why can't you just ` return certPool.getCertsPEM()` from `ValidateAndSplitPEMBundle`? Am I missing something? 🤔

Reply:
Oh, in that case we can! I thought you wanted to return `certPool`

Reply:
> Why can't you just `return certPool.getCertsPEM()` from `ValidateAndSplitPEMBundle`? Am I missing something? 🤔

I fixed a code as you required, all tests were passed.
Could you please review?